steam/ext/csgo/client.py

```diff
@@ -58,6 +62,64 @@ async def _handle_ready(self) -> None:
         self.http.user = ClientUser(self._connection, await self.http.get_user(self.user.id64))
         await super()._handle_ready()
 
+    @overload
+    async def inspect_item(self, *, owner: SteamID, asset_id: int, d: int) -> PreviewDataBlock:
+        ...
+
+    @overload
+    async def inspect_item(self, *, market_id: int, asset_id: int, d: int) -> PreviewDataBlock:
+        ...
+
+    @overload
+    async def inspect_item(self, *, url: str) -> PreviewDataBlock:
+        ...
+
+    async def inspect_item(
+            self,
+            *,
+            owner: SteamID | None = None,
+            asset_id: int = 0,
+            d: int = 0,
+            market_id: int = 0,
+            url: str = "",
+            _timeout: int = 10
```

Comment:
Why is this argument here if it's completely hidden?

Reply:
I just left him if user need more control over waiting timeout. Not sure, if this is ever come handy